packages/core/src/base/staticClient.ts

```diff
@@ -78,6 +79,25 @@ export function getHeader(
   return header;
 }
 
+/**
+ * Format the request being made for logging.
+ *
+ * @param resource The resource being requested
+ * @param fetchOptions The options to the fetch call
+ */
+export const formatFetchForInfoLog = 
+  (resource:string, fetchOptions:RequestInit): string => `Request: ${fetchOptions.method.toUpperCase()} ${resource}`;
+
+/**
+ * Format the response received for logging.
+ *
+ * @param response The response received
+ */
+export const formatResponseForInfoLog = (response:Response): string => {
+  const successString = (response.ok || response.status === 304) ? "successful" : "unsuccessful";
+  return `Received ${successString} response: ${response.status} ${response.statusText}`;
```

Comment:
Consider making it more uniform with the previous statement

```suggestion
  return `Response: ${successString} - ${response.status} ${response.statusText}`;
```

Or something like that

Reply:
I just copied the format from the AWS SDK 😹But I like yours, updated.